src/data_formatter.ts

```diff
@@ -240,3 +240,108 @@ export default class DataFormatter {
     }
   }
 }
+
+export function containsWideDataFrame(data: any[]) {
+  if (!data || !data.length) {
+    return false;
+  }
+
+  return data.some((d) => d && d.hasOwnProperty('type') && d.type === 'table');
+}
```

Comment:
this check seems to only check if it's a table, is this enough?

Reply:
I'm not sure but I could break this out to a separate function and add more logic like fields.length > 2 I guess?